i18n/locale-info.php

```diff
@@ -2559,7 +2559,7 @@
 		'currency_code'  => 'XPF',
 		'currency_pos'   => 'right_space',
 		'thousand_sep'   => ' ',
-		'decimal_sep'    => ',',
+		'decimal_sep'    => '.',
```

Comment:
The changes in this files should be consistent with https://github.com/woocommerce/woocommerce/pull/46960.